h2o-docs/src/product/explainability.rst

```diff
@@ -0,0 +1,213 @@
+
+- **exclude_explanations**: Exclude specified explanations.
+
+The available options (explainations) for ``include_explanations`` and ``exclude_explainations`` are:
+    
+    - ``"leaderboard"``
+    - ``"confusion_matrix"``
+    - ``"residual_analysis"``
+    - ``"variable_importance"``
+    - ``"variable_importance_heatmap"``
+    - ``"model_correlation_heatmap"``
+    - ``"shap_summary"``
+    - ``"pdp"``
+    - ``"ice"``
+
+- **top_n_features**: If ``columns_of_interest`` is missing, create plots only with the top n columns (where applicable).  Defaults to 5.
+
+- **plot_overrides**: Overrides for individual explanations, e.g. ``list(shap_summary_plot = list(top_n_features = 50))`` in R. 
+
+Notes
+~~~~~
+
+TO DO.
+
+
+Code Examples
+-------------
+
+TO DO.
+
+Explain models
+~~~~~~~~~~~~~~
+
+Here’s an example showing basic usage of the ``h2o.explain()`` function in *R* and the ``explain()`` method in *Python*.  
+
+.. tabs::
+   .. code-tab:: r R
+
+        library(h2o)
+        h2o.init()
+
+        # Import a sample binary outcome train/test set into H2O
+        train <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_train_10k.csv")
+        test <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_test_5k.csv")
+
+        # Identify predictors and response
+        y <- "response"
+        x <- setdiff(names(train), y)
+
+        # For binary classification, response should be a factor
+        train[, y] <- as.factor(train[, y])
+        test[, y] <- as.factor(test[, y])
+
+        # Run AutoML for 20 base models
+        aml <- h2o.automl(x = x, y = y, 
+                          training_frame = train,
+                          max_models = 20,
+                          seed = 1)
+
+        # Explain leader model & compare with all AutoML models                  
+        exa <- h2o.explain(aml, test)
+        print(exa)
+
+        # Explain a single H2O model (e.g. leader model from AutoML)
+        exm <- h2o.explain(aml@leader, test)
+        print(exm)
+
+
+
+   .. code-tab:: python
+
+        import h2o
+        from h2o.automl import H2OAutoML
+
+        h2o.init()
+
+        # Import a sample binary outcome train/test set into H2O
+        train = h2o.import_file("https://s3.amazonaws.com/erin-data/higgs/higgs_train_10k.csv")
+        test = h2o.import_file("https://s3.amazonaws.com/erin-data/higgs/higgs_test_5k.csv")
+
+        # Identify predictors and response
+        x = train.columns
+        y = "response"
+        x.remove(y)
+
+        # For binary classification, response should be a factor
+        train[y] = train[y].asfactor()
+        test[y] = test[y].asfactor()
+        
+        # Run AutoML for 20 base models
+        aml = H2OAutoML(max_models=20, seed=1)
+        aml.train(x=x, y=y, training_frame=train)
+
+        # Explain leader model & compare with all AutoML models 
+        exa = aml.explain(test)
+        print(exa)
```

Comment:
Remove `print` in python.

Reasons mentioned above.

Reply:
fixed